index.js

```diff
@@ -1,16 +1,18 @@
 'use strict';
+var PassThrough = require('stream').PassThrough;
 var Promise = require('pinkie-promise');
 
-function getStream(stream, opts) {
-	if (!stream) {
+function getStream(inputStream, opts) {
+	if (!inputStream) {
 		return Promise.reject(new Error('Expected a stream'));
 	}
 
-	opts = opts || {};
-	var encoding = opts.encoding || 'utf8';
+	var stream;
```

Comment:
Any reason for defining it here rather than inside the promise other than it looking better :)?